### PR TITLE
Add an addresses() method to EventLog to list related addresses

### PR DESCRIPTION
**Pull request by hansl**
This is generally more useful than verifying if a single event is related
to a single address.

### CODE REVIEW

**Comment by hansl** [comment]
@Denommus You will need this for the new Data attribute.

**Comment by Denommus** [comment]
CI is failing for this one.

File: src/many-modules/src/_4_events.rs [hunk 1]
```diff
@@ -637,6 +645,20 @@ mod test {
         assert_eq!(Into::<Vec<u8>>::into(t2), (v - 1).to_be_bytes());
     }
 
+    #[test]
+    fn event_info_addresses() {
```
Comment (fmorency) [inline src/many-modules/src/_4_events.rs:649]: Test inner resolution with a multisig submit tx.

**Review by fmorency — APPROVED** [review]
Thanks! Approving assuming comments are addressed/acknowledged.

nit: I can't comment directly on the review, but `[inner]` on line 512 is not properly aligned with the rest.

File: src/many-modules/src/_4_events.rs (hunk 1 above)
Comment (hansl) [reply src/many-modules/src/_4_events.rs:649]: We don't have resolution for `inner` yet. It's... very hard.

**Review by fmorency — APPROVED** [review]

**Pull request merged**